tiny_dnn/util/parallel_for.h

```diff
@@ -87,8 +87,9 @@ void parallel_for(size_t begin,
                   const Func &f,
                   size_t /*grainsize*/) {
   assert(end >= begin);
+  // unsigned index isn't allowed in OpenMP 2.0
 #pragma omp parallel for
-  for (size_t i = begin; i < end; ++i) f(blocked_range(i, i + 1));
+  for (int i = (int)begin; i < (int)end; ++i) f(blocked_range(i, i + 1));
```

Comment:
@edgarriba It is a restriction from openmp specification. See my comment and this:
https://stackoverflow.com/questions/2820621/why-arent-unsigned-openmp-index-variables-allowed